src/Quiz.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Drupal\anu_lms;
+
+use Drupal\anu_lms_assessments\Entity\AssessmentQuestionResult;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Quiz service.
+ */
+class Quiz extends Lesson {
+  const FIELD_NO_MULTIPLE_SUBMISSIONS = 'field_no_multiple_submissions';
+  const FIELD_QUESTION_RESPONSE = 'field_question_response';
+
+  /**
+   * Returns normalized data for Quiz page.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $node
+   *   Lesson node.
+   *
+   * @return array
+   *   An array containing current node and referenced course.
+   */
+  public function getPageData(EntityInterface $node) {
+    $data = parent::getPageData($node);
+    return $this->anuLmsGetQuizSubmissionData($node, $data);
+  }
+
+  /**
+   * Returns data for Quiz submissions.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $node
+   *   Quiz node.
+   * @param array $data
+   *   Data to be returned.
+   */
+  protected function anuLmsGetQuizSubmissionData(EntityInterface $node, array &$data) {
```

Comment:
no need to prefix methods with anuLms @esbaddeley

Reply:
changed